pages/common/exercism.md

```diff
@@ -0,0 +1,20 @@
+# exercism
+
+> Download and solve problems from the command-line.
+> More information: <https://exercism.org/docs/using/solving-exercises/working-locally>.
+
+- Configure application token and preferred workspace for Exercism:
+
+`exercism configure --token={{your-application-token}} --workspace={{/path/to/preferred/workspace}}`
+
+- Download a specific exercise:
```

Comment:
Is there no way to list exercises? If there is, that should be an example directly before this one.

Reply:
As per my knowledge, i do not think there is a way to list the exercises :(
We can `download/submit`